open-metadata-resources/open-metadata-labs/lineage/lineage.ipynb

```diff
@@ -0,0 +1,692 @@
+    "                \"elementTypeVersion\": 1,\n",
+    "                \"elementTypeDescription\": \"A set of properties describing a type of connector.\"\n",
+    "            },\n",
+    "            \"guid\": \"3851e8d0-e343-400c-82cb-3918fed81da6\",\n",
+    "            \"qualifiedName\": \"Kafka Open Metadata Topic Connector\",\n",
+    "            \"displayName\": \"Kafka Open Metadata Topic Connector\",\n",
+    "            \"description\": \"Kafka Open Metadata Topic Connector supports string based events over an Apache Kafka event bus.\",\n",
+    "            \"connectorProviderClassName\": \"org.odpi.openmetadata.adapters.eventbus.topic.kafka.KafkaOpenMetadataTopicProvider\",\n",
+    "            \"recognizedConfigurationProperties\": [\n",
+    "                \"producer\",\n",
+    "                \"consumer\",\n",
+    "                \"local.server.id\",\n",
+    "                \"sleepTime\"\n",
+    "            ]\n",
+    "        },\n",
+    "        \"endpoint\": {\n",
+    "            \"class\": \"Endpoint\",\n",
+    "            \"headerVersion\": 0,\n",
+    "            \"address\": \"server.cocoMDS4.omas.assetlineage.outTopic\"\n",
+    "        },\n",
+    "        \"configurationProperties\": {\n",
+    "            \"producer\": {\n",
+    "                \"bootstrap.servers\": \"lab-kafka:9092\"\n",
+    "            },\n",
+    "            \"local.server.id\": \"39b54be8-229f-4480-8a41-242b1c822669\",\n",
+    "            \"consumer\": {\n",
+    "                \"bootstrap.servers\": \"lab-kafka:9092\"\n",
+    "            }\n",
+    "        }\n",
+    "    },\n",
+    "    \"lineageGraphConnection\": {\n",
+    "        \"class\": \"Connection\",\n",
+    "        \"headerVersion\": 0,\n",
+    "        \"displayName\": \"Lineage Graph Connection\",\n",
+    "        \"description\": \"Used for storing lineage in the Open Metadata format\",\n",
+    "        \"connectorType\": {\n",
+    "            \"class\": \"ConnectorType\",\n",
+    "            \"headerVersion\": 0,\n",
+    "            \"connectorProviderClassName\": \"org.odpi.openmetadata.openconnectors.governancedaemonconnectors.openlineageconnectors.janusconnector.graph.LineageGraphConnectorProvider\"\n",
+    "        },\n",
+    "        \"configurationProperties\": {\n",
+    "            \"storage.directory\": \"data/servers/open-lineage/lineage-repository/berkeley\",\n",
+    "            \"index.search.directory\": \"data/servers/open-lineage/lineage-repository/searchindex\",\n",
+    "            \"storage.backend\": \"berkeleyje\",\n",
+    "            \"gremlin.graph\": \"org.janusgraph.core.JanusGraphFactory\",\n",
+    "            \"index.search.backend\": \"lucene\"\n",
+    "        }\n",
+    "    },\n",
+    "    \"accessServiceConfig\": {\n",
+    "        \"serverName\": \"cocoMDS4\",\n",
+    "        \"serverPlatformUrlRoot\": \"https://lab-datalake:9443\"\n",
```

Comment:
as with the event bus this needs to come from the relevant variable as we use in the config notebook, so that it works in all environments not just k8s